tests/acceptance/09_services/standard_services-from-non-default-namespace.cf

```diff
@@ -0,0 +1,50 @@
+body common control
+{
+      inputs => { "../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent test
+{
+  meta:
+      "description" -> { "ENT-5406" }
+        string => "Test that standard_services can be used from non-default namespace";
+
+  methods:
+      "Test" usebundle => ENT_5406:ns_test;
+}
+
+bundle agent standard_services(service, state) {
```

Comment:
What is this?

Reply:
It's default:standard_services, which is what services promises call by default if service_method is not specified.

Reply:
Note that in 3.15.x where the C code change has not been merged, this test is failing: https://github.com/cfengine/core/pull/4106 https://travis-ci.org/github/cfengine/core/builds/661594253?utm_source=github_status&utm_medium=notification

Reply:
So you're just using this dummy implementation to create a file. Initially I thought it would make more sense that you didn't add it, but rather used the one in masterfiles. Maybe add a comment?

Reply:
Right, because I don't actually want to try and start/stop services which is what the one from masterfiles would try to do.
I will add a comment